web/html/src/global.d.ts

```diff
@@ -47,6 +47,9 @@ declare global {
 
   // Used by section-toolbar, defined in spacewalk-essentials.js
   var handleSst: Function;
+
+  /** DEPRECATED: Do **NOT** use this global for new code, prefer `useUserLocalization()` instead */
+  var localTime: string | undefined;
 }
 
 export {};
```

Comment:
Not part of this PR but, could you please add a final empty line? Git/GitHub complain about it, thanks.